operations/service.go

```diff
@@ -97,7 +98,12 @@ func startSystemCronJobs(ctx context.Context, env evergreen.Environment) error {
 	////////////////////////////////////////////////////////////////////////
 	//
 	// Local Queue Jobs
-	amboy.IntervalQueueOperation(ctx, env.LocalQueue(), 30*time.Second, util.RoundPartOfMinute(0), opts, units.PopulateLocalQueueJobs(env))
+	local := env.LocalQueue()
+	amboy.IntervalQueueOperation(ctx, local, 30*time.Second, util.RoundPartOfMinute(0), opts, units.PopulateLocalQueueJobs(env))
+
+	// Enqueue jobs to ensure each app server has the correct SSH key files.
+	ts := util.RoundPartOfHour(30).Format(units.TSFormat)
+	grip.Error(local.Put(ctx, units.NewLocalUpdateSSHKeysJob(ts)))
```

Comment:
this will only run once (at some point after the application starts.) this seems like it's not quite enough? 

Also seems like it would be good to have more context in the logging message, and I'd be inclined to combine lines 105 and 106

Reply:
RIght now the admin settings only get updated in the DB and the changes won't apply unless the app server restarts, so this job runs once it sees the latest admin settings in the DB.

I wrapped the log message with some context. I don't want to combine L105-106 because having more than a few function calls on a single line because it just makes it painful to read.